runtime/lua/vscode-neovim/default-options.lua

```diff
@@ -17,4 +14,4 @@ function M.setup()
   vim.g.loaded_matchparen = 1
 end
 
-return M
```

Comment:
Why this pattern here, but not in filetype.lua?

Reply:
They are same

Reply:
Then keep it the same, there is value in consistency so a reader knows what to expect.